### DISCUSSION TITLE
Use Edges UDF instead of creating a new

### DISCUSSION THREAD

**Discussion by AHJKem** [MicrosoftEdge/WebView2Feedback#4385] [Q&A] [answered]
Hi,

I’m currently trying to point my environment to use Microsoft Edges UDF folder, is this possible? 

Here is an example of my code:

    Private Async Sub initializeWebpage()
            Dim UDF As String = "C:\Program Files (x86)\Microsoft\EdgeWebView\Application\121.0.2277.128"
            Dim WebPageEnvironment = CoreWebView2Environment.CreateAsync(browserExecutableFolder:=UDF, "C:\Program Files (x86)\Microsoft\EdgeWebView", Nothing)
            Await WebPage.EnsureCoreWebView2Async(WebPageEnvironment.Result)
    End Sub

I’m aware that you can specify a custom location however my goal is to not create an UDF. 

Could you please provide me with any assistance

**Comment by victorhuangwq** [comment]
@LiangTheDev could you take a look at this question?

**Comment by LiangTheDev** [comment, marked as answer]
It is not possible to user Edge browser's user data folder as WebView2s. We don't support sharing browsing context between WebView2s and browsers.

It seems that you have already read https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder?tabs=win32. If you are a packaged app, the default UDF is pretty good and you don't have to specify a custom UDF. Otherwise, you should specify a custom folder and remove the folder when the app is uninstalled. WebView2 needs to store files somewhere and the data stored could be sensitive.
